agent/api/container/container.go

```diff
@@ -266,6 +266,9 @@ type Container struct {
 	// the JSON body while saving the state
 	SteadyStateStatusUnsafe *apicontainerstatus.ContainerStatus `json:"SteadyStateStatus,omitempty"`
 
+	// ContainerArn is the Arn of this container.
+	ContainerArn string `json:"ContainerArn,omitempty"`
```

Comment:
do these changes (container.ContainerArn and taskLaunchType) need to be added to the state saved in boltdb as well?

Reply:
they will be automatically saved as part of task/container i think